### ISSUE TITLE
Forms are broken in 1.0.0.rc2

### ISSUE THREAD

**Issue by TheRPunkt** [mkhairi/materialize-sass#179]
Hello! 
I updated the gem today to the newest version (which should be 1.0.0.rc2).
Unfortunately I guess that the javascript that handles the label's active class does not work how it should.

This is my view code:
```    
= form_for @application, url: applications_path do |f|
    .input-field
        = f.text_field :name
        = f.label :name
 ```
(shortened)

This is how it looks for me:
![form_error](https://user-images.githubusercontent.com/16973216/43215206-cd94b14a-903b-11e8-97e0-20aaf1719a12.png)

As I can see the file size of the forms.js file has been decreased dramatically, maybe it has something to do with this (not sure though).

**Comment by mkhairi** [comment]
@TheRPunkt it should be ok as the [demo site](http://materialize.labs.my/forms) currently using 1.0.0.rc2 too.

**Comment by TheRPunkt** [comment]
@mkhairi Thanks for your quick reply. I did some research about my project and created a new plain rails project with only the materialize gem installed. With this done it actually worked. Afterwards I found out that I did one thing different:
In the test project I followed the installation instruction to require materialize while in my main project I required materialize-sprockets. After changing that it actually worked. 
However, I do not think that this is an intended behaviour, is it?
